src/backend/utils.ts

```diff
@@ -483,7 +485,7 @@ async function getSteamRuntime(
   const steamLibraries = await getSteamLibraries()
   const runtimeTypes: SteamRuntime[] = [
     {
-      path: 'steamapps/common/SteamLinuxRuntime_soldier/run',
+      path: 'steamapps//src/common/SteamLinuxRuntime_soldier/run',
```

Comment:
this change seems not intended (maybe a search/replace thing?)

Reply:
yup my mistake. fixed now